config/rubocop/.rubocop-layout.yml

```diff
@@ -0,0 +1,47 @@
+Layout/EmptyLinesAroundModuleBody:
```

Comment:
I find these configs a good default startinig point for rubocop. But these are preferential so we can tweak more moving forward.